bot/cogs/random_cog/random_cog.py

```diff
@@ -92,5 +93,44 @@ async def ball(self, ctx, *, question):
         embed = discord.Embed(title='🎱', description= f'{random.choice(responses)}',color = Colors.ClemsonOrange)
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['slotmachine','🎰'])
+    @commands.cooldown(1, SLOTS_COMMAND_COOLDOWN, commands.BucketType.guild)
+    async def slots(self, ctx):
+        """
+        A simple slot machine.
+            """
+        emojis = "🍎🍊🍐🍋🍉🍇🍓🍒"
+        a = random.choice(emojis)
+        b = random.choice(emojis)
+        c = random.choice(emojis)
+
+        blank = '⬜'
+
+        if (a == b ==c):
+            message = f'{ctx.message.author.mention} won!'
+        elif (a == b) or (a == c) or (b == c):
+            message = f'{ctx.message.author.mention} almost won, 2/3!'
+        else:
+            message = f'{ctx.message.author.mention} lost, no matches.'
+
+        embed = discord.Embed(title = '💎 Slot Machine 💎', color = Colors.ClemsonOrange, description = f'**{ctx.message.author.name} is rolling the slots**')
+        embed.add_field(name = f'{blank} | {blank} | {blank}', value = 'Spinning...', inline = False)
+        await ctx.send(embed = embed, delete_after = 1.75)
```

Comment:
we should edit the same embed instead of sending a new one, so that it remains the same logical place in chat

Reply:
We should also break that out into a function that is called to cut down on repeated code